podman_tutorial.md

```diff
@@ -0,0 +1,156 @@
+![PODMAN logo](https://cdn.rawgit.com/kubernetes-incubator/cri-o/master/logo/crio-logo.svg)
+
+# Basic Setup and Use of Podman
+Podman is a utility provided as part of the libpod library.  It can be used to create and maintain
+containers. The following tutorial will teach you how to set up Podman and perform some basic
+commands with Podman.
+
+## Install Podman on Fedora
+Many of the basic components to run Podman are readily available from the Fedora RPM repositories; the only
+exception is Podman itself.  In this section, we will help you install all the runtime and build dependencies
+for Podman until an RPM becomes available.
+
+### Installing build and runtime dependencies
+```
+# sudo dnf install -y git runc libassuan-devel golang golang-github-cpuguy83-go-md2man glibc-static \
+                                    gpgme-devel glib2-devel device-mapper-devel libseccomp-devel \
+                                    atomic-registries iptables skopeo-containers containernetworking-cni
+```
+### Building and installing podman
+```
+# git clone https://github.com/projectatomic/libpod/ ~/src/github.com/projectatomic/libpod
```

Comment:
I'd suggest some minor tweaks to be more like the Buildah install instructions.  Add these three lines above line 21:
```
mkdir ~/podman
cd ~/podman
export GOPATH=`pwd`
```
Then change ~/src/github... to ./src/github... on this line, then src/github... below to ./src/github... one line 22.  Regardless the two src/github... reference don't match and need to on lines 21 and 22.

Reply:
fixed